doc/migrate-to-motor-3.rst

```diff
@@ -0,0 +1,461 @@
+| j                  | journal                       | -                                                      |
++--------------------+-------------------------------+--------------------------------------------------------+
+| wtimeout           | wTimeoutMS                    | -                                                      |
++--------------------+-------------------------------+--------------------------------------------------------+
+
+MotorClient.fsync is removed
+............................
+
+Removed :meth:`~motor.motor_tornado.MotorClient.fsync`. Run the
+`fsync command`_ directly with :meth:`~motor.motor_tornado.MotorDatabase.command`
+instead. For example::
+
+    await client.admin.command('fsync', lock=True)
+
+.. _fsync command: https://mongodb.com/docs/manual/reference/command/fsync/
+
+MotorClient.unlock is removed
+.............................
+
+Removed :meth:`~motor.motor_tornado.MotorClient.unlock`. Run the
+`fsyncUnlock command`_ directly with
+:meth:`~motor.motor_tornado.MotorDatabase.command` instead. For example::
+
+     await client.admin.command('fsyncUnlock')
+
+.. _fsyncUnlock command: https://mongodb.com/docs/manual/reference/command/fsyncUnlock/
+
+
+MotorClient.max_bson_size/max_message_size/max_write_batch_size are removed
+...........................................................................
+
+Removed :attr:`~motor.motor_tornado.MotorClient.max_bson_size`,
+:attr:`~motor.motor_tornado.MotorClient.max_message_size`, and
+:attr:`~motor.motor_tornado.MotorClient.max_write_batch_size`. These helpers
+were incorrect when in ``loadBalanced=true mode`` and ambiguous in clusters
+with mixed versions. Use the `hello command`_ to get the authoritative
+value from the remote server instead. Code like this::
+
+    max_bson_size = client.max_bson_size
+    max_message_size = client.max_message_size
+    max_write_batch_size = client.max_write_batch_size
+
+can be changed to this::
+
+    doc = await client.admin.command('hello')
+    max_bson_size = doc['maxBsonObjectSize']
+    max_message_size = doc['maxMessageSizeBytes']
+    max_write_batch_size = doc['maxWriteBatchSize']
+
+.. _hello command: https://mongodb.com/docs/manual/reference/command/hello/
+
+MotorClient.event_listeners and other configuration option helpers are removed
+..............................................................................
+
+The following client configuration option helpers are removed:
+- :attr:`~motor.motor_tornado.MotorClient.event_listeners`.
+- :attr:`~motor.motor_tornado.MotorClient.max_pool_size`.
+- :attr:`~motor.motor_tornado.MotorClient.min_pool_size`.
+- :attr:`~motor.motor_tornado.MotorClient.max_idle_time_ms`.
+- :attr:`~motor.motor_tornado.MotorClient.local_threshold_ms`.
+- :attr:`~motor.motor_tornado.MotorClient.server_selection_timeout`.
+- :attr:`~motor.motor_tornado.MotorClient.retry_writes`.
+- :attr:`~motor.motor_tornado.MotorClient.retry_reads`.
```

Comment:
This list doesn't render: https://motor--166.org.readthedocs.build/en/166/migrate-to-motor-3.html#motorclient-event-listeners-and-other-configuration-option-helpers-are-removed

Reply:
fixed